src/nvim/api/vim.c

```diff
@@ -207,18 +208,38 @@ String nvim_replace_termcodes(String str, Boolean from_part, Boolean do_lt,
   return cstr_as_string(ptr);
 }
 
-String nvim_command_output(String str, Error *err)
+/// Executes an ex-command and returns its (non-error) output.
+/// Shell |:!| output is not captured.
+///
+/// On parse error: forwards the Vim error; does not update v:errmsg.
+/// On runtime error: forwards the Vim error; does not update v:errmsg.
```

Comment:
Are these two intended to be different? Otherwise can be simplified

Reply:
I plan to review all API functions again to nail down the differences.